client/src/pages/Settings/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,44 @@
+import { makeStyles } from '@mui/styles';
+import { Theme } from '@mui/material/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  flexContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '100%',
+  },
+  profilePhoto: {
+    height: 200,
+    width: 200,
+    borderRadius: '50%',
+  },
+  picText: {
+    width: '40%',
+    fontSize: 14,
+    textAlign: 'center',
+    color: 'lightgray',
+  },
+  submit: {
+    margin: theme.spacing(3, 2, 2),
+    padding: 10,
+    width: 250,
+    height: 56,
+    borderRadius: theme.shape.borderRadius,
+    fontSize: 16,
+    backgroundColor: 'white !important',
```

Comment:
It is not recommended to use `!important` to override MUI styles. You can instead use on of the methods from this article: https://blog.bitsrc.io/4-ways-to-override-material-ui-styles-43aee2348ded